server.js

```diff
@@ -1,31 +1,121 @@
 import express from 'express'
 import bodyParser from 'body-parser'
 import cors from 'cors'
+import booksData from './data/books.json'
```

Comment:
Next time let's remember about one line break between outer and local dependencies ;)